java/refactoring.java/test/unit/src/org/netbeans/modules/refactoring/java/ui/JavaRefactoringActionsProviderTest.java

```diff
@@ -66,12 +67,18 @@ public void show(RefactoringUI ui, TopComponent activetc) {
         assertEquals(++expectedCount, called.get());
     }
 
+    // hg blame: "#190101: preventing exceptions from various refactorings for very broken sources (without a top-level class)"
```

Comment:
Maybe there should be a GH Issue for this (although we have many GH issues, of course).